Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  SignInViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/02/26.
+//
+import UIKit
+
+protocol Validation {
+    func isValidCredentialsUsername(username: String) -> Bool
+    func isValidCredentialsPassword(password: String) -> Bool
+}
+
+let bottomLine = CALayer()
+let bottomLine2 = CALayer()
+
+class SignInViewController: UIViewController, Validation {
+    func isValidCredentialsUsername(username: String) -> Bool {
+        if username == "Admin"{
+            return true
+        } else {
+            return false
+        }
+    }
+
+    func isValidCredentialsPassword(password: String) -> Bool {
+        if password == "TestPass123" {
+            return true
+        } else {
+            return false
+        }
+    }
+    @IBOutlet weak var txtUsername: UITextField!
+    @IBOutlet weak var txtPassword: UITextField!
+    @IBOutlet weak var btnSignIn: UIButton!
+    @IBOutlet weak var btnRegister: UIButton!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addCornerRadius(btnSignIn)
+        addCornerRadius(btnRegister)
+        changeBorderLook(btnRegister)
+        addUnderline(txtUsername, txtPassword, bottomLine: bottomLine, bottomLine2: bottomLine2)
+    }
+
+    @IBAction private func btnSignInClick(_ sender: UIButton!) {
```

Comment:
So just some more naming enhancements here: 

Normally we don't like to user abbreviations as well as since we are doing mobile development tapped is a little bit more appropriate: 

`signInButtonTapped`